test/pytest/conftest.py

```diff
@@ -22,29 +22,42 @@ def any_subprocesses() -> bool:
         return True  # at least one process (or zombie) still waitable
 
 
-@pytest.fixture(autouse=True)
-def event_loop(monkeypatch) -> Iterable[asyncio.AbstractEventLoop]:
-    loop = EventLoopPolicy().new_event_loop()
+if sys.version_info < (3, 7, 0):
+    # Polyfills for Python 3.6 (plus compat with pre-event_loop_policy pytest-asyncio versions)
```

Comment:
Why do we still need to bother about py 3.6 here? We don't support RHEL 8 from the main branch any more, and have dropped the CentOS 8 unit tests long ago.

Reply:
I want to make sure the bridge Python code keeps working on RHEL 8 for beibooting.  At the present time, tox is the only thing keeping us honest at all....